### ISSUE TITLE
Natural language label.

### ISSUE THREAD

**Issue by Kushagra7777** [Seqaeon/EMNIST_streamlit#1]
Hi Pearse @Seqaeon !
Thank you for your EMNIST effort. The application works really well, I really like how you are loading EMNIST dataset through torch. But I'd like to raise an issue pointed by @mi3law. 
''In the Agent Visual Inspector section, the "Actual Label" displays an integer value, which appears to represent the class of the image. However, displaying the actual natural language class (example: 'pi', 'sigma', 'negative sign', 'check', 'plus') in place of actual label would make more sense."  
We believe, EMNIST provides this information in their dataset. 
We'd appreciate if you can look into it.

**Comment by Seqaeon** [comment]
Yeah, I thought about that too. Will do that as soon as possible. Thanks @Kushagra7777

**Comment by Seqaeon** [comment]
Done @Kushagra7777  and @mi3law

**Comment by Kushagra7777** [comment]
Thank you. We will look into it.

**Comment by Seqaeon** [comment]
How is it? @Kushagra7777

**Comment by Kushagra7777** [comment]
@Seqaeon We are testing it and will let you know the updates. 
Thank you

**Comment by Seqaeon** [comment]
Hello @Kushagra7777 

I seem to have ran into some issue while trying to play around with ao_core that I cant figure what is causing it. (Image atatched)

![image](https://github.com/user-attachments/assets/2e1689b3-5a84-4f51-8693-332b17b429ff)


Also how goes the testing?

**Comment by Seqaeon** [comment]
The same code seemed to be working a few days ago.

**Comment by Seqaeon** [comment]
Any Updates on this @Kushagra7777
